activation/poet.go

```diff
@@ -104,8 +104,10 @@ func (r retryableHttpLogger) Warn(format string, args ...any) {
 	r.inner.Sugar().Warnw(format, args...)
 }
 
+// Debug seems to be the only logging level used by retryablehttp. Since it logs when it retries a request,
+// and we want users to see what is happening we change the level to Info.
 func (r retryableHttpLogger) Debug(format string, args ...any) {
-	r.inner.Sugar().Debugw(format, args...)
+	r.inner.Sugar().Infow(format, args...)
 }
```

Comment:
Does it become spammy? IDK how much retryablehttp logs. If it does, perhaps it could check if `format` contains a string like "retrying" etc., wdyt?

Reply:
Good idea. I checked https://github.com/hashicorp/go-retryablehttp/blob/v0.7.7/client.go and the only thing that is logged on debug by the the http client is "performing request" (once per request) and "retrying request" (once per retry) those would be the only additional log messages we see wit that change.

I think thats not too many log messages so we don't need an additional filter, we can try it for now and see if it becomes a problem.